{"last_activity_date": 1477451290, "owner": {"display_name": "jals", "reputation": 3, "profile_image": "https://www.gravatar.com/avatar/1bab675b8b78704fb327e8ed1c3e2320?s=128&d=identicon&r=PG&f=1", "user_id": 6913775, "user_type": "registered", "link": "http://stackoverflow.com/users/6913775/jals"}, "is_answered": false, "answer_count": 0, "creation_date": 1477451290, "score": 0, "title": "Django QuerySet union throws TypeError", "tags": ["python", "django", "django-queryset"], "view_count": 25, "question_id": 40253043, "link": "http://stackoverflow.com/questions/40253043/django-queryset-union-throws-typeerror"}